app/controllers/organizations/staff_controller.rb

```diff
@@ -1,6 +1,9 @@
 class Organizations::StaffController < Organizations::BaseController
   layout "dashboard"
 
+  before_action :organization_admin
```

Comment:
```suggestion
  before_action :require_organization_admin
```
I think this makes it a bit more informative about what this before_action does